web/app/js/components/MetricsTable.jsx

```diff
@@ -130,6 +126,8 @@ const columnDefinitions = (sortable = true, resource, namespaces, onFilterClick,
 
   if (resource.toLowerCase() === "namespace") {
```

Comment:
Would it be possible to strip this down a little bit? I don't see any reason why we can't show meshed traffic on the namespace page and it will get rid of some of the logic.

```
  if (!showNsColumn) {
    return _.concat(columns, percentMeshedColumn);
  } else {
    return _.concat(nsColumn, columns, percentMeshedColumn);
  }
```

(Obviously, this'd move percentMeshedColumn into the columns list).